pkg/knode-manager/adapters/k8s/help.go

```diff
@@ -0,0 +1,97 @@
+package k8sadapter
+
+import (
+	"strings"
+
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/klog"
+)
+
+func getSecrets(pod *corev1.Pod) []string {
+	secretNames := []string{}
+	for _, v := range pod.Spec.Volumes {
+		switch {
+		case v.Secret != nil:
+			if strings.HasPrefix(v.Name, "default-token") {
+				continue
+			}
+			klog.Infof("pod %s depends on secret %s", pod.Name, v.Secret.SecretName)
+			secretNames = append(secretNames, v.Secret.SecretName)
+
+		case v.CephFS != nil:
+			klog.Infof("pod %s depends on secret %s", pod.Name, v.CephFS.SecretRef.Name)
+			secretNames = append(secretNames, v.CephFS.SecretRef.Name)
+		case v.Cinder != nil:
+			klog.Infof("pod %s depends on secret %s", pod.Name, v.Cinder.SecretRef.Name)
+			secretNames = append(secretNames, v.Cinder.SecretRef.Name)
+		case v.RBD != nil:
+			klog.Infof("pod %s depends on secret %s", pod.Name, v.RBD.SecretRef.Name)
```

Comment:
We can leave out this part of the logic for now and let the developers from the open-source community who join later add it

Reply:
For now, let's do this

Reply:
we do not need it now